src/components/Header.jsx

```diff
@@ -18,6 +20,9 @@ function Header() {
           </NavLink>
         </li>
       </ul>
+      <div className="userOval">
+        <img className="userAvatar" src={userAvatar} alt="" />
+      </div>
```

Comment:
- [ ] I like your design, everything is cool. But according to the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Your%20final%20design%20should%20match%20Zeplin%27s%20design.), your design should match to original  design. Kindly update it. Font size, font weight positioning of elements, and layouts should be EXACTLY the same as the original design.
- [ ] Also make sure website is fullscreen like in the original design. Maybe you can see white space left and right side on it kindly ignore it. [Design link](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Login%20to%20Zeplin%20(hint%3A%20you%20will%20find%20the%20credentials%20in%20the%20sneak%20peek%20of%20the%20project).)
![image](https://user-images.githubusercontent.com/38635158/192045908-05e369cf-0bd6-48f7-bbea-e9f255da6ea2.png)
___
Original 
![image](https://user-images.githubusercontent.com/38635158/192045940-37e1f7c3-37aa-4939-84d2-d407c08a54e8.png)